.github/pull_request_template.md

```diff
@@ -1,20 +1,21 @@
-## Context
+### Context
 
-<!-- Why are you making this change? What might surprise someone about it? -->
+Include relevant motivation and context.
 
-## Changes proposed in this pull request
+### Changes proposed in this pull request
 
-<!-- If there are UI changes, please include Before and After screenshots. -->
+Include a summary of the change.
 
-## Guidance to review
+### Guidance to review
 
-<!-- How could someone else check this work? Which parts do you want more feedback on? -->
+Include any useful information needed to review this change.
+Include any dependencies that are required for this change.
 
-## Link to Trello card
```

Comment:
Worth keeping this I think — useful to be able to click through to the card from the PR.